### ISSUE TITLE
deserialization errors because of seemingly harmless files in config store

### ISSUE THREAD

**Issue by rkoblenzer** [interactive-instruments/ldproxy#210]
A file like `.../data/store/entities/services/test.yml.bak` with a harmless content like this:
```
# test
```

... should not lead to an error in the logfile:

```
INFO  [2020-08-19 14:26:40,006] Started web server at http://localhost:7080/
ERROR [2020-08-19 14:26:45,497] Could not deserialize entities Identifier{id=test.yml, path=[services]}, format 'bak' unknown.
```

This is not urgent. It seems that the error has no side effects, and there is a work-around: renaming the file to an invalid ID (e.g. `!test.yml`) ensures that it will be ignored. 

**UPDATE**: With a file `_info_` containing just a short description ldproxy is not starting at all. The last message is `Started web server at http://localhost:7080/`, after this, I cannot see any progress or activity.

**Comment by azahnen** [comment]
Should be fixed by #212. 

Files with unknown extensions are now silently ignored.

Files without extension but with a name that is a valid id, like `_info_`, are intepreted as JSON. That is required for backwards compatibility. An error will be logged if such a file is not valid JSON, but startup will proceed normally. 

We plan to remove this migration path in the next minor release.

**Comment by rkoblenzer** [comment]
For now it is fixed, but after 2.0 we should discuss the handling of non-configuration files in general, see #interactive-instruments/xtraserver-webapi/issues/69.
